### DISCUSSION TITLE
Network v13 'HyperDrive'

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#155] [Network + Implementations Announcements]
This is for discussing the upcoming [Network v13 'HyperDrive'](https://github.com/filecoin-project/community/discussions/74#discussioncomment-707228)🚀!

**Comment by Jasonwu123** [comment]
What time to upgrade?

**Reply by Fatman13** [reply]
+1

**Reply by rjan90** [reply]
From the announcement page: `Tentatively: Mid-June`
https://github.com/filecoin-project/community/discussions/74#discussioncomment-707228

**Reply by diwufeiwen** [reply]
It takes about 2 days for venus to merge the new changes，the version release time is kept in sync with lotus.

**Reply by Fatman13** [reply]
> delay the HyperDrive upgrade epoch by a week to early July

**Comment by TheDivic** [comment]
Hey guys!

The `v13` update is really promising, great job. 👏 

I was wondering do you have any _metrics_, _tests_, or at least _estimates_ in place to measure if the upcoming update really delivers the improvement in storage onboarding rates, bandwidth, and gas fees?

**Reply by rjan90** [reply]
For some estimates: https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md#batch-incentive-alignment

**Reply by TheDivic** [reply]
Great, thanks!
